src/main/resources/processes/TO_QA.bpmn

```diff
@@ -75,116 +78,160 @@
     <bpmn:sequenceFlow id="Flow_0n7rdkp" sourceRef="Gateway_0vour6f" targetRef="TO_QA_OUTCOME">
       <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${QaStatus == "Save"}</bpmn:conditionExpression>
     </bpmn:sequenceFlow>
+    <bpmn:userTask id="TO_GET_CAMPAIGN_TYPE" name="Get Campaign Type" camunda:formKey="TROF_CAMPAIGN_REQUEST">
+      <bpmn:incoming>Flow_1l1f1dy</bpmn:incoming>
+      <bpmn:outgoing>Flow_1j2kqxk</bpmn:outgoing>
+    </bpmn:userTask>
+    <bpmn:sequenceFlow id="Flow_1l1f1dy" sourceRef="Gateway_0vour6f" targetRef="TO_GET_CAMPAIGN_TYPE">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${QaStatus == "PutOnCampaign"}</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
+    <bpmn:exclusiveGateway id="Gateway_05qknzm" name="Direction?" default="Flow_1sjleue">
+      <bpmn:incoming>Flow_1j2kqxk</bpmn:incoming>
+      <bpmn:outgoing>Flow_1ekh7v9</bpmn:outgoing>
+      <bpmn:outgoing>Flow_1sjleue</bpmn:outgoing>
+    </bpmn:exclusiveGateway>
+    <bpmn:sequenceFlow id="Flow_1j2kqxk" sourceRef="TO_GET_CAMPAIGN_TYPE" targetRef="Gateway_05qknzm" />
+    <bpmn:sequenceFlow id="Flow_1ekh7v9" sourceRef="Gateway_05qknzm" targetRef="TO_QA_OUTCOME">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${ DIRECTION == "BACKWARD" }</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
+    <bpmn:sequenceFlow id="Flow_1sjleue" sourceRef="Gateway_05qknzm" targetRef="Event_0nlva02" />
```

Comment:
This flow also should not be going straight to the end, rather to the `Update Approval: DecisionNotYetMade` process task.